src/pages/Facility/services/inventory/internalTransfer/ToDispatchSupplyRequestTable.tsx

```diff
@@ -139,28 +140,29 @@ export default function ToDispatchSupplyRequestTable({
                 {Object.values(SupplyRequestStatus)
                   .filter((status) => status !== SupplyRequestStatus.draft)
                   .map((status) => (
-                    <CommandItem
-                      key={status}
-                      value={status}
-                      onSelect={() =>
-                        updateQuery({
-                          status:
+                    <PopoverClose key={status} className="w-full">
```

Comment:
Don't mix radix's popover with ours, can cause some weird behaviors.

Instead wrap with regular popover from ui/components, similar to how it's done in PurchaseOrders.tsx